### PR TITLE
Add AttributeMatcher

### PR DESCRIPTION
**Pull request by maksimf**

### CODE REVIEW

File: CHANGES.txt [hunk 1]
```diff
@@ -1,6 +1,7 @@
 4.3.0 (<ADD DATE>)
 - Add impression listener
 - Add support for client shutdown (destroy())
+- Add AttributeMatcher
```
Comment (patricioe) [inline CHANGES.txt]: we are actually adding support to apply attribute matcher to the traffic type

File: NEWS [hunk 2]
```diff
@@ -1,6 +1,6 @@
 4.3.0
 
-Add support for impression listener, there is an ability to use user-defined class to capture all impressions
+Add support for impression listener, there is an ability to use user-defined class to capture all impressions, add AttributeMatcher
```
Comment (patricioe) [inline NEWS]: let's do one line per topic.
second line should be: "Now you can apply attribute matcher to the traffic type "

**Comment by maksimf** [comment]
@patricioe fixed

**Comment by maksimf** [comment]
@patricioe Sure, will do

**Comment by maksimf** [comment]
@patricioe Done

**Pull request merged**